life_expectancy/cleaning.py

```diff
@@ -1,133 +1,110 @@
 """Module to clean assignment data"""
+
 import os
 import re
 import argparse
 import pandas as pd
 from pandas import DataFrame
 
 
-def load_raw_data(
-    raw_data_file_path: str
-):
+def load_data():
     """
     This function is responsible for loading the raw data,
     applying the correct split and tranforming it to a pandas dataframe
 
-    param: raw_data_file_path -> path to the raw data file
-
-    return: Pandas dataframe with columns properly defined
+    return: eu_life_expectancy_raw_dataframe -> pandas dataframe
     """
-    os_path_pardir = os.path.dirname(__file__)
+
+    #Get parent directory
+    parent_directory = os.path.dirname(__file__)
+    raw_data_file_path = '/data/eu_life_expectancy_raw.tsv'
```

Comment:
Paths shall be defined for the module scoped and not function scoped. It can introduce code smells.

Path of the file shall not be hardcoded, as well